ckanext/scheming/helpers.py

```diff
@@ -339,6 +339,8 @@ def scheming_display_json_value(value, indent=2):
         serialized.
     :rtype: string
     """
+    if not value:
+        return value
```

Comment:
I don't think this fits here. the function description would have to become much more complicated. "returns a JSON string, or an empty string, or [] or {} or False or None.." Can we move this check into the snippet instead?

Reply:
What about renaming to `scheming_display_json_object` and checking that `isinstance(value, dict)` and if not returning value? This goes on the same direction of having the preset only dealing with objects.

Reply:
On second thoughts the snippet is meant to be generic, so I'll do what you suggest and handle it there